tasks/rh-sign-image/tests/test-rh-sign-image-multiple-components.yaml

```diff
@@ -111,5 +114,6 @@ spec:
                   exit 1
                 fi
               done
+              kubectl delete internalrequests --all
```

Comment:
This should be in a finally. If it is needed to cleanup, we should cleanup even if the test fails I would think. See https://github.com/redhat-appstudio/release-service-catalog/blob/main/tasks/collect-data/tests/test-collect-data.yaml#L153 for an example

Reply:
fixed